src/sortStudents.ts

```diff
@@ -1,16 +1,67 @@
 
 export interface Student {
-  // describe Student interface
+  name: string;
+  surname: string;
+  age: number;
+  married: boolean;
+  grades: number[];
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+function sortAverageGrade(element: number[]): number {
```

Comment:
Pay attention to variables naming
```suggestion
function getAverageGrade(grades: number[]): number {
```